code/game/gamemodes/objective.dm

```diff
@@ -2,17 +2,28 @@ GLOBAL_LIST(admin_objective_list) //Prefilled admin assignable objective list
 GLOBAL_LIST_EMPTY(objectives)
 
 /datum/objective
-	var/datum/mind/owner //The primary owner of the objective. !!SOMEWHAT DEPRECATED!! Prefer using 'team' for new code.
-	var/datum/team/team //An alternative to 'owner': a team. Use this when writing new code.
-	var/name = "generic objective" //Name for admin prompts
-	var/explanation_text = "Nothing" //What that person is supposed to do.
+	///The primary owner of the objective. !!SOMEWHAT DEPRECATED!! Prefer using 'team' for new code.
```

Comment:
Can you elaborate this more in your comment? It won't be obvious to people (including me) why I would want a solo-objective to be under a team, and not a mind.

Reply:
I literally have no idea what these old comments are smoking, i'm not even sure what to say

Reply:
We should really tear all this down

Reply:
oh i didnt realize you didnt make this